tests/conftest.py

```diff
@@ -1674,12 +1674,40 @@ def tmpvenv_active_inherit(deactive_venv, tmpdir: py.path.local) -> Iterator[env
     loader.unload_modules_for_path(venv.site_packages_dir)
 
 
+@pytest.fixture(scope="session")
+def create_local_package_index_factory():
```

Comment:
Let's put the word "empty" explicitly in the name. Otherwise we get into a naming conflict if we ever want to have a factory for non-empty indexes.
```suggestion
def create_empty_local_package_index_factory():
```